### PR TITLE
Changelog Entry + doc.go File + Basic Samples

### PR DESCRIPTION
**Pull request by tasherif-msft**

### CODE REVIEW

File: sdk/storage/azqueue/README.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+[azure_sub]: https://azure.microsoft.com/free/
+[azidentity]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity
+[storage_ad]: https://docs.microsoft.com/azure/storage/common/storage-auth-aad
+[azcore_response_error]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azcore#ResponseError
+[samples]: https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/storage/azqueue/samples_test.go
+[queue_error]: https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/storage/azqueue/queueerror/error_codes.go
+[queue]: https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/storage/azqueue/queue_client.go
+[sas]: https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/storage/azqueue/sas
+[service]: https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/storage/azqueue/service_client.go
+[storage_contrib]: https://github.com/Azure/azure-sdk-for-go/blob/main/CONTRIBUTING.md
```
Comment (jhendrixMSFT) [inline sdk/storage/azqueue/README.md:239]: References `azblob`

File: sdk/storage/azqueue/README.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+you to agree to a Contributor License Agreement (CLA) declaring that you have
+the right to, and actually do, grant us the rights to use your contribution. For
+details, visit [cla.microsoft.com][cla].
+
+This project has adopted the [Microsoft Open Source Code of Conduct][coc].
+For more information see the [Code of Conduct FAQ][coc_faq]
+or contact [opencode@microsoft.com][coc_contact] with any
+additional questions or comments.
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-go%2Fsdk%2Fstorage%2Fazblob%2FREADME.png)
```
Comment (jhendrixMSFT) [inline sdk/storage/azqueue/README.md:217]: References `azblob`

File: sdk/storage/azqueue/go.mod [hunk 3]
```diff
@@ -4,16 +4,24 @@ go 1.18
 
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.2.0
```
Comment (jhendrixMSFT) [inline sdk/storage/azqueue/go.mod:6]: Might be good to run `go get -u` to update these dependencies.

File: sdk/storage/azqueue/README.md (hunk 2 above)
Comment (tasherif-msft) [reply sdk/storage/azqueue/README.md:217]: its just a PNG, I think ill just remove this impressions tag altogether

File: sdk/storage/azqueue/README.md (hunk 1 above)
Comment (jhendrixMSFT) [reply sdk/storage/azqueue/README.md:239]: The dangers of "find all"

Comment (tasherif-msft) [reply sdk/storage/azqueue/README.md:239]: I'm still glad you're double checking, blobs was def a big point of reference haha

**Review by jhendrixMSFT — APPROVED** [review]

**Comment by tasherif-msft** [comment]
linter currently fails since the links in the README are referencing main branch rather than this branch. This should be fixed once everything is merged into main.

**Review by vibhansa-msft — COMMENTED** [review]
LGTM

**Pull request merged**